src/UI/templates/default/Button/toggle.less

```diff
@@ -0,0 +1,33 @@
+.il-toggle-button {
+  position: relative;
+  //display: inline-block;
+  height: 19px;
+  padding: 0px 25px;
+  border: 1px solid rgba(0,0,0,0.1);
+  border-radius: 40px;
+  background: lightgrey;
+  cursor: pointer;
+}
+
+.il-toggle-button .il-toggle-switch {
+  position: absolute;
+  top: -4px;
+  left: -6%;
+  width: 25px;
+  height: 25px;
+  border: 1px solid rgba(0,0,0,0.2);
+  border-radius: 40px;
+  background: grey;
+
+  -webkit-transition: all 0.25s ease-in-out; /* Safari */
+  transition: all 0.25s ease-in-out;
+}
+
+.il-toggle-button.on {
+  background: #b3b3cc;
+}
+
+.il-toggle-button.on .il-toggle-switch {
+  left: 53%;
+  background: #666699;
```

Comment:
Where does this come from? This color is unknown to ilias afaik. Probably use a variation of 6ea03c defined as own variable for the toogle.